### ISSUE TITLE
[Accessibility: Resize text] Unable to change guest permission when zoom 200%

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/kimchi#895]
There is no way to add/remove users and groups to a guest.
Only the first panel is displayed.

![353e794cd63109f6](https://cloud.githubusercontent.com/assets/4921718/14019617/756458b0-f1b2-11e5-8134-e0515f153f26.jpg)
